src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx

```diff
@@ -283,7 +283,7 @@ Framework-specific configurations:
 
 ### Infrastructure monitoring [#infrastructure-endpoints]
 
-If you have infrastructure agent version [1.15.0 or higher](/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1150), simply enable the [FedRAMP configuration option](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings#fedramp). 
+If you have infrastructure agent version [1.15.0 or higher](/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1150), simply enable the [FedRAMP configuration option](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings#fedramp). This enables FedRAMP compliancy for data reported by the infrastructure agent, and for any [on-host integrations](/docs/integrations/host-integrations/host-integrations-list) that work with the infrastructure agent to report data. 
```

Comment:
In Gatsby build "On host integrations list" appears twice in TOC. I sent you a screenshot to show you what I mean.